examples/kotlin-diktat/warn/save.toml

```diff
@@ -0,0 +1,20 @@
+[general]
+tags = warn
+description = "Test warnings discovered by diKTat"
+suiteName = "warnings"
+ignoreSaveComments = true
+
+[warn]
+execFlags = --debug  # todo: replace with empty string once ktoml supports quotes
```

Comment:
Is it just placeholder, while you're writing, that we need to replace it with empty string in future?
May be we can just just don't specify `execFlags` at all? Validation method should change `null` to the empty string.

Reply:
I originally added it because we got an NPE if no `execFlags` have been specified. I believe we have fixed it already, so I'll remove it from here.